cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -34,31 +34,65 @@
 
    ========================================================================== #}
 
-{% macro render( settings, options={} ) %}
-
-{% if settings.heading and settings.body and settings.image_url %}
+{% if value %}
 <section class="hero"
          {{ 'style=background-color:' +
-             options.background_color + ';'
-             if options.background_color else '' }}>
+             value.background_color + ';'
+             if value.background_color else '' }}>
     <div class="wrapper hero_wrapper">
         <div class="hero_text">
-            <h1 class="hero_heading">{{ settings.heading }}</h1>
-            <p class="hero_subhead">{{ settings.body }}</p>
-            {% if options.link_text and options.link_url %}
-                <a href="{{ options.link_url }}"
+            <h1 class="hero_heading">{{ value.heading | safe}}</h1>
+            <p class="hero_subhead">{{ value.body |richtext}}</p>
+            {% if value.link_text and value.link_url %}
+                <a href="{{ value.link_url }}"
                    class="hero_cta
-                   {{ 'btn' if options.is_button
+                   {{ 'btn' if value.is_button
```

Comment:
It's in the [Design Manual Spec](http://stephanieosan.github.io/design-manual/ui-toolkit/heroes.html) as "Call to Action Button"